src/renderer/videoJS.css

```diff
@@ -1383,11 +1387,11 @@ body.vjs-full-window {
   position: relative;
   top: -1px;
 }
-
-.video-js .vjs-play-control .vjs-icon-placeholder {
+/*
+.video-js .vjs-play-control .vjs-icon-placeholder {spinner
```

Comment:
Is it crashing because of the added "spinner"?

Reply:
sorry just caught that extra reference to 'spinner' that shouldn't be there - FreeTube is now fine